modules/server/src/mapping/resolveAggregations.ts

```diff
@@ -26,7 +59,7 @@ export default ({ type, getServerSideFilter }) => {
 			nestedFieldNames,
 			filters: compileFilter({
 				clientSideFilter: resolvedFilter,
-				serverSideFilter: getServerSideFilter(context),
+				serverSideFilter: getServerSideFilter(),
```

Comment:
this doesn't appear to be needed? As far as I read, es client is only thing in context and the filter ops shouldn't be used directly in that. thoughts? - Jon, Justin